bin/git-utimes

```diff
@@ -25,6 +33,12 @@ else
   # because all args will go into single worker.
   NPROC=$(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null || getconf _NPROCESSORS_ONLN 2>/dev/null)
   # shellcheck disable=SC2086
-  git ls-tree -z -r -t --name-only @ \
-  | xargs -0 -P${NPROC:-1} -n${NPROC:-1} $opt_r git utimes --touch
+  # don't touch files that have been modified in the worktree or index
+  #   bsd doesn't have `-z` option for `comm` and `cut`, so use `tr` as work around
+  prefix="$(git rev-parse --show-prefix) "
+  comm -23 <(git ls-tree -z -r --name-only --full-name @ | tr '\0' '\n') \
```

Comment:
I get `comm: file 1 is not in sorted order` when running the command. Maybe something is missing?

Reply:
OK, I had thought that git ls-tree was always sorted... Let me fix it and add your other suggestions and re-commit.